pennylane/devices/default_qubit_jax.py

```diff
@@ -145,6 +145,8 @@ def circuit():
             and variances analytically. In non-analytic mode, the ``diff_method="backprop"``
             QNode differentiation method is not supported and it is recommended to consider
             switching device to ``default.qubit`` and using ``diff_method="parameter-shift"``.
+            Or keeping ``default.qubit.jax`` but switching to
+            ``diff_method=qml.gradient.stoch_pulse_grad`` for pulse programming.
```

Comment:
Note that it is `qml.gradients` (i.e., plural)

Reply:
Fixed in #4004